IronFrame.Shared/Utilities/JobObject.cs

```diff
@@ -291,9 +314,12 @@ private void SetJobLimits(NativeMethods.JobObjectExtendedLimitInformation extend
 
                 Marshal.StructureToPtr(extendedLimit, extendedInfoPtr, false);
 
-                if (!NativeMethods.SetInformationJobObject(handle, NativeMethods.JobObjectInfoClass.ExtendedLimitInformation, extendedInfoPtr, length))
+                if (
```

Comment:
Are you using Resharper?  There are a lot of unnecessary formatting changes in this PR that makes it difficult to see the actual functionality changes.  Also, some of the changes (like this one) do not improve the formatting IMO.  It might be a good idea to turn off the auto-formatting to reduce noise in PRs.

Re: style in general, I'm totally open to style changes, but we should probably make them as separate commits.